oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -416,6 +416,12 @@ Apache-ShenYu:
 Hutool:
   id: 128
   languages: Java
+MicronautHttpServer:
+  id: 129
+  languages: Java
+MicronautHttpClient:
+  id: 130
+  languages: Java
```

Comment:
1. **129** has been assigned, which is why I asked you to register this ID first.
2. If you want to have two IDs, one for the client, and the other for the server, then you need to set up a mapping to make sure if the agent is installed only on Client-server, the virtual node of server could have the correct icon.

Reply:
The ID only used for mapping icon ?

Reply:
Not just that, it also represents the component of spans. If you use the same, the client and server spans are going to show the same component.
If you need two, choose 130 and 131, and set the mapping at the bottom of this file.

Reply:
OK, I choose 130 and 131

Reply:
Sure.